src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/deliuscontext/StaffDetail.kt

```diff
@@ -16,14 +18,23 @@ data class StaffDetail(
     .sortedByDescending { it.startDate }
 
   fun teamCodes() = teams.map { it.code }
+
+  fun toStaffMember() = StaffMember(
+    staffCode = this.code,
+    staffIdentifier = this.staffIdentifier,
+    forenames = this.name.forenames(),
+    surname = this.name.surname,
+    username = this.username,
+  )
 }
 
 data class PersonName(
   val forename: String,
   val surname: String,
-  val middleName: String?,
+  val middleName: String? = null,
 ) {
   fun deliusName() = "$forename $surname"
+  fun forenames() = "$forename ${middleName?.takeIf { it.isNotEmpty() } ?: ""}".trim()
```

Comment:
Do we need this function? I think we use full name which also include the surname

Reply:
I mean if we have forename, middleName and surname. I think the only method that is need is the full name with include all three properties. 

I am not sure if the forenames in the StaffMember includes also the middleName